### DISCUSSION TITLE
Tìm 1 câu tục ngữ/ thành ngữ/ ca dao hoặc thơ bất kì

### DISCUSSION THREAD

**Discussion by VuThamPhi** [VuThamPhi/Lop-10A06-Nhom-09#11] [Ideas]
câu của Phi là:
"Có công mài sắt, có ngày nên kim."

**Comment by VuThamPhi** [comment]
Mọi người tìm 1 câu rồi nhóm chốt chọn 1 câu để làm

**Comment by VuThamPhi** [comment]
Tui phân công việc luôn nha:
1. giới thiệu 
2. giải thích câu ca dao/ tục ngữ...
3. ý nghĩa của câu ca dao/ tục ngữ...
4. rút ra bài học qua câu ca dao/ tục ngữ...
5. kết luận, khẳng định lại 
Có 5 phần vậy theo tui thì 1 bạn làm số 1 và 5 1 bạn làm số 2 1 bạn làm số 3 1 bạn làm số 4 nha.

**Comment by longphan090306** [comment]
> câu của Phi là: "Có công mài sắt, có ngày nên kim."

Vậy là mình thống nhất câu này đúng không

**Reply by VuThamPhi** [reply]
Vậy mình chốt câu này đi

**Comment by 40ThanhTrung** [comment]
Mọi người tìm thêm mấy dẫn chứng về câu ca dao tục ngữ " Có công mài sắt có ngày nên kim" đi nha
Của t là: Ai cũng biết tới vợ chồng hai nhà khoa học người Pháp Pierre Curie và Marie Curie. Suốt 4 năm ròng rã, họ đã kì công lọc đi lọc lại vô số lần trong 8 tấm quặng để tìm 1/10 gram chất phóng xạ radium. Qua việc phát hiện ra một nguyên tố hóa học, chúng ta mới phần nào hình dung ra sự kiên trì bền bỉ vô cùng mãnh liệt khi nghiên cứu phát minh một thành tựu phục vụ xã hội loài người.

**Reply by VuThamPhi** [reply]
Dẫn chứng của tui là:
Những tấm gương chăm học, những tấm gương chịu khó như Bác Hồ là một điển hình rõ nét nhất. Bác đã phải vất vả làm việc, chịu khó học tiếng nước ngoài, đi bôn ba khắp nơi để tìm đường cứu nước. Và cũng nhờ những sự nỗ lực đó mà đất nước ta mới được tự hào về một danh nhân, một vị lãnh tụ vĩ đại nổi tiếng mà khắp năm, châu bốn bể đều biết tới.

**Reply by nguyenducuy** [reply]
Dẫn chứng của tui là:
Có những tấm gương sáng ngời và tiêu biểu của câu có công mài sắc có ngày nên kim.Trong số đó phải kể đến tấm gương vượt khó của 2 bạn học sinh Minh Hiếu Và Tất Minh đã cõng nhau đi học suốt 10 năm trời.Cùng với sự nổ lực không ngừng nghỉ và ý chí quyết tâm nên đôi bạn ấy đã trở thành học sinh giỏi tỉnh và thi tốt nghiệp trên 28 điểm và đậu vào trường đại học mà mình mong muốn.
